packages/frontend/src/features/scheduler/components/SchedulerModalContent.tsx

```diff
@@ -109,6 +109,17 @@ const CreateStateContent: FC<{
     return (
         <SchedulerForm
             disabled={createMutation.isLoading}
+            resource={
+                isChart
+                    ? {
+                          uuid: resourceUuid,
+                          type: 'chart',
+                      }
+                    : {
+                          uuid: resourceUuid,
+                          type: 'dashboard',
+                      }
```

Comment:
I don't like this but I also don't like having `isChart` propagated. does not feel typesafe

Reply:
🤷 TBH I can't think of a better way than a context and a provider for this 😅